src/components/tracked-controls.js

```diff
@@ -18,7 +17,13 @@ module.exports.Component = registerComponent('tracked-controls', {
     controller: {default: 0},
     id: {type: 'string', default: ''},
     idPrefix: {type: 'string', default: ''},
-    rotationOffset: {default: 0}
+    rotationOffset: {default: 0},
+    // Arm model parameters, to use when not 6DOF. (pose hasPosition false, no position)
+    headElement: {type: 'selector'},
+    hand: {type: 'string', default: 'right'},
+    eyesToElbow: {default: {x: 0.175, y: -0.3, z: -0.03}}, // vector from eyes to elbow (divided by user height)
+    forearm: {default: {x: 0, y: 0, z: -0.175}}, // vector from eyes to elbow (divided by user height)
```

Comment:
those are scaled by user height, yes.  but there need to be base values to utilize.  they are in the schema in case one wants to override to better match something else being modeled e.g. robot with longer or shorter arms